### ISSUE TITLE
Bug: Line graph shows a data marker even when no data exists for that point

### ISSUE THREAD

**Issue by nkittur-uga** [VEuPathDB/web-eda#1411]
Issue identified by users during the TMRC workshop (10/12/2022)

To recreate: 
SCORE S. mansoni Cluster Randomized Trial
Filter: Country=Tanzania, Village study arm= Arm 3 
Make a Line Plot with the parameters as in screenshot below

<img width="1215" alt="Screen Shot 2022-10-13 at 1 45 30 PM" src="https://user-images.githubusercontent.com/67710160/195668535-07a134a2-d0a2-4721-b256-a4f7a37466da.png">

Notice that the line plot shows markers at Y=0 for Year 3 and Year 4. In fact, there was no data for these years as these were treatment holiday years (see screenshot below showing no participants for these years).

<img width="544" alt="Screen Shot 2022-10-13 at 1 45 12 PM" src="https://user-images.githubusercontent.com/67710160/195668531-cdc8ae4e-3708-4430-a47a-734b3f1c04c4.png">

There should be no markers when there is no data.

**Comment by danicahelb** [comment]
Discussed in 12/2/2022 EDA UX meeting. Markers at 0 when y-axis proportions are 0/0 are confusing to users. This was brought up by participants during the TMRC workshop in October

For continuous variables, we do NOT show markers at 0 when there is no data

The birds eye view & variable completeness table exist so that missing data is accounted for

**Decision**: do not add markers at 0 when the proportion is 0/0

**Comment by danicahelb** [comment]
This ticket essentially duplicates https://github.com/VEuPathDB/web-eda/issues/1504

**Comment by moontrip** [comment]
@danicahelb Yes, I thought so: that was the reason why I gathered them by myself as next cycle 😃 Thank you for handling this 👍

**Comment by aaronwlsong** [comment]
Recreated the example, looks good now, closing ticket.



![Image](https://user-images.githubusercontent.com/61077845/211399349-2048e754-50fb-4db5-8cdb-f5c17141e74d.png)

**Comment by danicahelb** [comment]
This ticket has had off-target affects on markers for undefined proportions (0/0), so I am reopening it.

See this example from LLINEUP (unfiltered)

**QA site looks like this: (birds eye view shows 115,542 data points but not all are represented on the plot since 0/0 markers are missing):**
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/25450900/224828165-5209f1a2-ce54-4e5a-947a-7596c59446d8.png">

**Live site looks like this:**
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/25450900/224828418-7250b828-838d-4eae-8eaf-b93e0c3a7f08.png">

**Comment by chowington** [comment]
Per https://github.com/VEuPathDB/EdaNewIssues/issues/602#issuecomment-1468655446, I'm considering this issue to still need UX discussion
